identity_stardust/src/did_or_placeholder.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::str::FromStr;
+
+use identity_core::common::KeyComparable;
+use identity_did::did::BaseDIDUrl;
+use identity_did::did::CoreDID;
+use identity_did::did::DIDError;
+use identity_did::did::DIDUrl;
+use identity_did::did::DID;
+use serde::Deserialize;
+use serde::Serialize;
+
+lazy_static::lazy_static! {
+  pub static ref PLACEHOLDER_DID: CoreDID =
+  CoreDID::parse("did:0:0").unwrap();
+}
```

Comment:
We should switch to `once_cell`. Apparently that's the preferred replacement for `lazy_static` now (news to me too).